en/docs/what-is-choreo.md

```diff
@@ -10,7 +10,7 @@ The following are some of the cutting-edge capabilities of Choreo:
 
  - Efficient and effective development:
     - **Code in any language**: Develop applications in your team's preferred programming language, harnessing existing skill sets to build powerful polyglot APIs, services, events, tasks, and more.
-    - **Native support for VSCode and GitHub**: Seamlessly collaborate and manage code with VSCode and GitHub in a feature-rich and visually-enhanced development environment.
+    - **Native support for VS Code and GitHub**: Seamlessly collaborate and manage code with VS Code and GitHub in a feature-rich and visually-enhanced development environment.
```

Comment:
Should we add GitLab and BitBucket as well?

Reply:
Added **Git repository integration** as a separate point.